pkg/pebinaryexecutor/pebinary.go

```diff
@@ -100,7 +100,7 @@ func (p *PEBinaryConfig) Bootstrap(ctx context.Context, nodeConfig *config.Node,
 	if err != nil {
 		return err
 	}
-	if err := os.Setenv("PATH", execPath+":"+os.Getenv("PATH")); err != nil {
+	if err := os.Setenv("PATH", execPath+";"+os.Getenv("PATH")); err != nil {
```

Comment:
FYI there is `os.PathListSeparator` if you want to avoid forgetting which is which on different platforms.

https://pkg.go.dev/os#pkg-constants